quesma/clickhouse/parser.go

```diff
@@ -256,6 +266,13 @@ func RemoveTypeMismatchSchemaFields(m SchemaMap, t *Table) SchemaMap {
 		}
 	}
 	for fieldName, v := range m {
+		withoutDotsFieldName := strings.Replace(fieldName, ".", "::", -1)
+		if fieldName != withoutDotsFieldName {
+			m[withoutDotsFieldName] = v
+			delete(m, fieldName)
+			fieldName = withoutDotsFieldName
+		}
+
```

Comment:
Isn't this portion of the code now unnecessary since you also modified `BuildInsertJson` (which calls this function, but after it already fixed all `.`s)?

Reply:
I can refactor to one.